components/IntentProperty.js

```diff
@@ -1,4 +1,5 @@
 import React, { PropTypes } from 'react'
+import $ from 'jquery'
```

Comment:
ah I miss them.
jquery would be included with bootstrap, but it will be better to put it in package.json